Project.toml

```diff
@@ -0,0 +1,16 @@
+name = "NGSIM"
+uuid = "195157bc-cedd-5875-9ea7-7cfb75f30b3c"
+version = "0.0.0"
+
+[deps]
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
```

Comment:
Should we add Project.toml dependencies for the unregistered packages too? I might have done that wrong in other repos if we aren't supposed to.

Reply:
I could add them for consistency, I think it is better to have them here, but it forces to check in the Manifest.toml for travis to pass.